miditok/classes.py

```diff
@@ -576,6 +577,12 @@ class TokenizerConfig:
         ``10``)
     :param ac_repetition_track_num_consec_bars: number of successive bars to
         compare the repetition similarity between bars. (default: ``4``)
+    :param add_trailing_bars: will add tokens for trailing empty Bars, if they are
```

Comment:
Ideally this should be added as a REMI-only parameter as it only affects this tokenizer. I recommend to move this documentation portion in REMI's docstring, and follow what is done for the `use_bar_end_tokens` argument given in `**kwargs` of the `TokenizerConfig`.

Reply:
Yes, that is good point, I can place add_trailing_bars there, but it seems like currently there is no place to document additional_params...

Reply:
https://miditok--204.org.readthedocs.build/en/204/tokenizations.html#miditok.REMI
see description for additional_params here